lib/matplotlib/backends/__init__.py

```diff
@@ -10,8 +10,7 @@
 # ipython relies on interactive_bk being defined here
 from matplotlib.rcsetup import interactive_bk
 
-__all__ = ['backend','show','draw_if_interactive',
-           'new_figure_manager', 'backend_version']
+__all__ = list(map(str, ['backend', 'pylab_setup']))
```

Comment:
What is the point of doing `list(map(str, ['foo', 'bar']))`?
On May 30, 2015 10:42 AM, "Thomas A Caswell" <notifications@github.com>
wrote:

> In lib/matplotlib/backends/__init__.py
> <https://github.com/matplotlib/matplotlib/pull/4486#discussion_r31381201>:
>
> > @@ -10,8 +10,7 @@
> >  # ipython relies on interactive_bk being defined here
> >  from matplotlib.rcsetup import interactive_bk
> >
> > -__all__ = ['backend','show','draw_if_interactive',
> > -           'new_figure_manager', 'backend_version']
> > +__all__ = list(map(str, ['backend', 'pylab_setup']))
>
> I don't think things should not be removed from this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4486/files#r31381201>.
>